plugin.py

```diff
@@ -119,7 +121,44 @@ def install_or_update(cls) -> None:
             shutil.rmtree(cls.basedir(), ignore_errors=True)
             raise
 
-    # notification handlers
+    # -- commands from the server that should be handled client-side ----------
+
+    def on_pre_server_command(
+        self,
+        command: Mapping[str, Any],
+        done_callback: Callable[[], None]
+    ) -> bool:
+        name = command["command"]
+        if name == "omnisharp/client/findReferences":
+            return self._handle_quick_references(
+                command["arguments"],
+                done_callback
+            )
+        return False
+
+    def _handle_quick_references(
+        self,
+        arguments: List[Any],
+        done_callback: Callable[[], None]
+    ) -> bool:
+        session = self.weaksession()
+        if not session:
+            return False
+        sb = session.get_session_buffer_for_uri_async(arguments[0]["uri"])
+        if not sb:
+            return False
+        for sv in sb.session_views:
+            if not sv.view.is_valid():
+                continue
+            r = Range.from_lsp(arguments[0]["range"])
+            region = range_to_region(r, sv.view)
+            args = {"point": region.a}
+            session.window.run_command("lsp_symbol_references", args)
```

Comment:
Can we not run this command on the view since it's a TextCommand?

Also, is `arguments[0]["uri"]` always the currently open view? Because if not then this code won't work, I think?